src/dmd/dinterpret.d

```diff
@@ -47,6 +47,21 @@ import dmd.tokens;
 import dmd.utf;
 import dmd.visitor;
 
+
+void incUsageCtfe(InterState* istate, const ref Loc loc)
```

Comment:
This belongs in the `private` section of this file.

Reply:
does matter.

Reply:
Would be nice to retain the current structure of this file (entry points up front, implementation later).

You can include it with other changes.

Reply:
That structure argument is good one.
I'll change it

Reply:
Done